lib/miq_automation_engine/models/miq_ae_datastore.rb

```diff
@@ -110,10 +110,10 @@ def self.import_yaml_dir(dirname, domain, tenant)
     _log.info("Import from #{dirname}...Complete - Benchmark: #{t.inspect}")
   end
 
-  def self.export
+  def self.export(tenant)
     require 'tempfile'
     temp_export = Tempfile.new('ae_export')
-    MiqAeDatastore.backup('zip_file' => temp_export.path, 'overwrite' => true)
+    MiqAeDatastore.backup('zip_file' => temp_export.path, 'overwrite' => true, 'tenant' => tenant)
```

Comment:
@kbrock 
No this is the tenant that came in from the controller that needs to be passed into export